PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,14 @@
 # Contributor Comments
 
-
-
 ## Pull Request Checklist
 
 Thank you for submitting a contribution to our project!
 
 In order to streamline the review of your contribution we ask that you review
 and comply with the below requirements:
 
-- [ ] Rebase your branch against the latest commit of the target branch
-- [ ] If you are adding a dependency, please explain how it was chosen
-- [ ] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results
+- [ ] Rebase your branch against the latest commit of the target branch.
+- [ ] If you are adding a dependency, please explain how it was chosen.
+- [ ] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results.
 - [ ] If there is an issue associated with your Pull Request, link the issue to the PR.
+- [ ] If a feature is being added, removed, or changed, code changes must be accompanied by the appropriate documentation updates or additions.
```

Comment:
Added new PR requirement

Reply:
Instead of the `If` approach, what if we just describe the desired outcome?

```suggestion
- [ ] Validate that documentation is accurate and aligned to any project updates or additions.
```

Reply:
I was leaning into consistent sentence structure, but this is cleaner and more to the point. I like it.